frontend/ui/elements/menu_activate.lua

```diff
@@ -55,7 +54,21 @@ return {
                 UIManager:show(InfoMessage:new{
                     text = _("This will take effect on next restart."),
                 })
-            end
+            end,
+            separator = true,
+        },
+        {
+            text = _("Show bottom menu"),
+            checked_func = function()
+                return G_reader_settings:readSetting("show_bottom_menu") ~= false
```

Comment:
G_reader_settings:nilOrTrue("show_bottom_menu")
(would return true if the setting is nil or true :) false otherwise (that is, when it is false)